.github/workflows/windows_build_test_and_push.yml

```diff
@@ -31,6 +31,50 @@ jobs:
        git -c submodule."external/commonItems".update=none `
            submodule update --init --recursive external\Fronter
 
+    - name: "Generate semantic version number"
+      uses: paulhatch/semantic-version@v5.0.2
+      id: version_number
+      with:
+        # The prefix to use to identify tags
+        tag_prefix: "v"
+        # A string which, if present in a git commit, indicates that a change represents a
+        # major (breaking) change, supports regular expressions wrapped with '/'
+        major_pattern: "(MAJOR)"
+        # A string which indicates the flags used by the `major_pattern` regular expression. Supported flags: idgs
+        major_regexp_flags: ""
+        # Same as above except indicating a minor change, supports regular expressions wrapped with '/'
+        minor_pattern: "(MINOR)"
+        # A string which indicates the flags used by the `minor_pattern` regular expression. Supported flags: idgs
+        minor_regexp_flags: ""
+        # A string to determine the format of the version output
+        version_format: "v${major}.${minor}.${patch}"
+        # If this is set to true, *every* commit will be treated as a new version.
+        bump_each_commit: true
+        # If true, the body of commits will also be searched for major/minor patterns to determine the version type.
+        search_commit_body: false
+        # The output method used to generate list of users, 'csv' or 'json'.
+        user_format_type: "csv"
+
+    - name: "Echo version number"
+      run: |
+       echo ${{ steps.version_number.outputs.major }}
+       echo ${{ steps.version_number.outputs.minor }}
+       echo ${{ steps.version_number.outputs.patch }}
+
+    - name: "Create version file"
```

Comment:
I wonder if there will be a problem here because the file already exists, it may need to be removed first.

Reply:
That's why the first line only has a single >, that overwrites it.